www/source/docs/best-practices.html.md.erb

```diff
@@ -303,35 +303,186 @@ Chef Habitat packages may be exported with the Supervisor directly into a [a var
+[Kubernetes](http://kubernetes.io/) is an open source container cluster manager that is available as a stand-alone platform or embedded in several distributed platforms including [Google's Container Engine](https://cloud.google.com/container-engine/), [AWS Elastic Kubernetes Service](https://aws.amazon.com/eks/), [Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/), and [Red Hat OpenShift](https://openshift.com/). Chef Habitat and Kubernetes are complementary: Kubernetes focuses on providing a platform for deployment, scaling, and operations of application containers across clusters of hosts while Chef Habitat manages the build pipeline and lifecycle of those application containers.
 
-### Chef Habitat Operator
+## Chef Habitat on Kubernetes
 
-The [Chef Habitat Kubernetes Operator](https://github.com/habitat-sh/habitat-operator) is on-going work to create an operator that leverages Kubernetes API services to create a native and robust integration between the two technologies.
+Chef Habitat can export Docker containers that run on Kubernetes. To benefit from Chef Habitat's gossip ring features like Service Discovery (binds) and Secrets, you must have a Chef Habitat bastion pod. A bastion pod provides an [Initial Peer](/docs/best-practices/#robust-supervisor-networks) to all other pods.
 
-By using the Chef Habitat Operator, you can abstract from many of the low level details of running a Chef Habitat package in Kubernetes, and jump straight to deploying your application, with support for Chef Habitat features like [service configuration](https://github.com/habitat-sh/habitat-operator/tree/master/examples/config), [binding](https://github.com/habitat-sh/habitat-operator/tree/master/examples/bind), [topologies](https://github.com/habitat-sh/habitat-operator/tree/master/examples/leader) and more.
+The bastion pod is deployed in a robust fashion with a Kubernetes Stateful Set, Persistent Volume and livenesss checking to ensure availability and ring data persistence. A Kubernetes Service is attached to the Stateful Set in order to make it discoverable with DNS. It is recommended to deploy one Service and Stateful Set combination per Namespace.
 
-For more details on the Chef Habitat Operator, please refer to the [introductory blog post](https://kinvolk.io/blog/2017/10/habitat-operator---running-habitat-services-with-kubernetes/), follow along on [github](https://github.com/habitat-sh/habitat-operator), and join us in our [#kubernetes](https://habitat-sh.slack.com/messages/kubernetes/) channel in the [Chef Habitat Slack](https://slack.habitat.sh).
+### Deploy the Chef Habitat bastion on Kubernetes
 
-### Kubernetes exporter
+Complete examples may be downloaded from [this folder](https://www.habitat.sh/docs/examples/kubernetes_hab_bastion/)
```

Comment:
is this the right URL/path for the examples?  @kagarmoe

Reply:
It needs to go into www/source/data. I've opened a PR to add the directory.
Blocked by https://github.com/habitat-sh/habitat/pull/7145

Reply:
@irvingpop I've re-thought how to deal with this. Let's put this example into the `www/source/partials/docs/` folder. I'll do the work to make it visible in a different PR once this merges. Thanks

Reply:
hi @kagarmoe I moved the examples folder to that location.  Do I need to change the URL in the link on this line or is that okay as-is?

Reply:
Let's remove the line until I get the examples wired in. Could you make an issue to add this line back in?

Reply:
Lets do it as-is and I'll make a card to fix it.